js/src/lib/Details/index.js

```diff
@@ -170,13 +178,21 @@ class Details extends Component {
             this.props.objectID
           )}
         </h2>
-        <p>{window.i18n.detail.not_found.yours}</p>
+        <p>
+          {window.i18n.detail.not_found.yours}
```

Comment:
These are `prettier` changes, probably because this file hasn’t been changed since last version update or something.